blocks/init/src/Blocks/custom/columns/styles/_shared.scss

```diff
@@ -30,7 +30,7 @@ $block-columns: (
 		}
 	}
 
-	&__vertical-spacing {
+	&__verticalSpacing {
```

Comment:
To me this looks like a problem in how the class name is generated in. We don't write SCSS with camel case 👀